pootle/core/views.py

```diff
@@ -632,6 +634,9 @@ def get_context_data(self, *args, **kwargs):
         ctx, cookie_data = self.sidebar_announcements
         ctx.update(
             super(PootleBrowseView, self).get_context_data(*args, **kwargs))
+
+        language_code, project_code = split_pootle_path(self.pootle_path)[:2]
```

Comment:
@phlax can you shed light on this https://github.com/translate/pootle/pull/4696#discussion-diff-64014273 ?
`self.language` can be a dict.